src/main/java/com/SAS/soccer_association_system/transaction/Transaction.java

```diff
@@ -0,0 +1,98 @@
+package com.SAS.soccer_association_system.transaction;
+
+import java.time.LocalDate;
+
+/**
+ * The class represent a transaction that can be made by a team owner or manger
+ */
+public class Transaction {
+
+    private float amount;
+    private transactionType type;
+    private LocalDate date;
+    private Team team;
+
+    /**
+     * Empty constructor
+     */
+    public Transaction() {
```

Comment:
I think the transaction should have only constructor with parameters, otherwise there's no meaning for it.

Reply:
I will delete it.